src/main/webapp/js/index.js

```diff
@@ -81,18 +81,13 @@ function submissionCounter(currentDate, baseDate, submissionPerHour, baseCount)
     if (!currentDate || !baseDate) {
         return errorMsg;
     }
-    var CurrBaseDateDifference = new Date(currentDate - baseDate);
+    var CurrBaseDateDifference = currentDate - baseDate;
```

Comment:
Just to check, is there a reason why the first letter of <code>CurrBaseDateDifference </code> is capitalised? Could we make a new issue to fix this, or fix it here?